### ISSUE TITLE
Portrait video cannot be update to 0 degrees rotation

### ISSUE THREAD

**Issue by SamStephens** [OpenShot/openshot-qt#4207]
**Describe the bug:**

When loading a Portrait video, the video loads with rotation set to 90 degrees, which is correct because the video is recorded in Landscape with rotation set to 90 degrees. This confirms https://github.com/OpenShot/openshot-qt/issues/1281 is fixed, I was checking for that when I found this issue.

The problem is however that I cannot then set the rotation to 0 degrees, if I do then want to use the video in Landscape for some reason. I can set the rotation to any value at all other than 0. You can work around this issue by setting the rotation to 0.0000001 degrees.

**Steps to reproduce the behavior:**

1. Load a portrait video (one that is stored landscape with the rotation metadata set to 90 degrees, as most phones do).
2. Put that video on the timeline
3. Attempt to update the properties of the clip so the rotation is 0 degrees
4. The rotation will be 90 degrees.

**Expected behavior:**

I expect to be able to set rotation to 0 degrees.

**System Details:**
- OpenShot Version [e.g. 2.4.3]: OpenShot-v2.5.1-dev3-daily
- Operating System / Distro: Windows 10

**Log Files:**
N/A

**Exception / Stacktrace:**
N/A

**Comment by JacksonRG** [comment]
That's really interesting! I'll take a look at this. Probably just a misplaced minimum value.

**Comment by JacksonRG** [comment]
Just out of curiosity have you tried setting it to 360?

**Comment by SamStephens** [comment]
Just confirmed I can set it to 360, but not to 0. Also confirmed that landscape video starts with a rotation of 0, and can be changed from 90 degrees and back to 0 again.

**Comment by JacksonRG** [comment]
@SamStephens Thanks for all this help! This really narrows down where I need to look.

**Comment by JacksonRG** [comment]
Quck fix: I found that set a rotation keyframe of 0 degrees at the *end* of the clip, it will give you rotation zero.

Still working on why `0` turns into `90` in every other case. 
I have found that when you `right click -> rotation -> no rotation`. So I think we're misinterpreting '0' degrees as 'the rotation of the file but everywhere else we're setting the rotation in absolute terms.

Now I just need to make it consistent whether the rotation is added/subtracted from the file's rotation, or just overridden.
